articles/service-fabric/service-fabric-containers-volume-logging-drivers.md

```diff
@@ -164,11 +165,13 @@ In the **Volume** element in the snippet above, the Azure Files volume plugin re
 
 As shown in the **DriverOption** elements in the snippet above, the Azure Files volume plugin supports the following driver options:
 
+Required driver option:
 - **shareName** - Name of the Azure Files file share that provides the volume for the container
 - **storageAccountName** - Name of the Azure storage account that contains the Azure Files file share
 - **storageAccountKey** - Access key for the Azure storage account that contains the Azure Files file share
 
-All of the above driver options are required.
+Optional driver option:
+- **storageAccountFQDN** - Domain Name of the Azure Files file share. If storageAccountFQDN is not in the options, domain name will be either storageAccountName if it represents FQDN or formed by using the default suffix(.file.core.windows.net) with the storageAccountName

```

Comment:
Since storage account FQDN is a concept that we have invented and it is not defined anywhere in Azure storage documentation, we should give an example otherwise the user may not understand exactly what they need to specify here. Please consider giving a public Azure example and a private cloud (e.g. Mooncake) example. The latter is important because this feature is mostly added for private cloud users.